### PR TITLE
Windows Virtual Desktop Helper: Add version 1.16

### PR DESCRIPTION
**Pull request by dankrusi**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] Manifest tested (install, uninstall, update)
- [x] AutoUpdate installed

Scoop package initial commit for Scoop Extras (by community request) (via https://github.com/dankrusi/WindowsVirtualDesktopHelper/issues/86)

### CODE REVIEW

File: bucket/windows-virtualdesktop-helper.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "url": "https://github.com/dankrusi/WindowsVirtualDesktopHelper/releases/download/v$version/WindowsVirtualDesktopHelper.Executable.v$version.zip",
+        "hash": {
+            "url": "https://github.com/dankrusi/WindowsVirtualDesktopHelper/releases/download/v$version/WindowsVirtualDesktopHelper.Executable.v$version.zip.sha256"
+        }
+    },
+    "## Comment 1": "https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests",
+    "## Comment 2": "https://github.com/ScoopInstaller/Scoop/wiki/Creating-an-app-manifest",
+    "## Comment 3": "https://github.com/ScoopInstaller/Extras",
+    "## Internal Comment 4": "https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate",
+    "## Internal Comment 5": "https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md"
```
Comment (aliesbelik) [inline bucket/windows-virtualdesktop-helper.json:29]:
```suggestion
    }
```

File: bucket/windows-virtualdesktop-helper.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        [
+            "WindowsVirtualDesktopHelper.exe",
+            "Windows Virtual Desktop Helper"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/dankrusi/WindowsVirtualDesktopHelper/releases/download/v$version/WindowsVirtualDesktopHelper.Executable.v$version.zip",
+        "hash": {
+            "url": "https://github.com/dankrusi/WindowsVirtualDesktopHelper/releases/download/v$version/WindowsVirtualDesktopHelper.Executable.v$version.zip.sha256"
```
Comment (aliesbelik) [inline bucket/windows-virtualdesktop-helper.json:22]:
```suggestion
            "url": "$url.sha256"
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
Please remove comments.
Otherwise looks good.

File: bucket/windows-virtualdesktop-helper.json (hunk 2 above)
Comment (dankrusi) [reply bucket/windows-virtualdesktop-helper.json:22]: thnx

**Review by aliesbelik — APPROVED** [review]

**Comment by dankrusi** [comment]
@aliesbelik thnx

**Pull request merged**